bears/rust/RustClippyLintBear.py

```diff
@@ -0,0 +1,82 @@
+import json
+import shutil
+import os
+
+from coalib.bears.GlobalBear import GlobalBear
+from coalib.misc.Shell import run_shell_command
+from coalib.results.Result import Result
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+
+
+class RustClippyLintBear(GlobalBear):
+    LANGUAGES = {'Rust'}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {
+            'Formatting',
+            'Unused Code',
+            'Syntax',
+            'Unreachable Code',
+            'Smell',
+            'Code Simplification'
+    }
+    EXECUTABLE = 'cargo.exe' if os.name == 'nt' else 'cargo'
```

Comment:
You might be interested in https://gitlab.com/coala/package_manager/issues/55 , which would allow us to manage the dependency like we do with other languages, and the `check_prerequisites` would be unnecessary.